src/components/header/AppHeader.vue

```diff
@@ -0,0 +1,536 @@
+<template>
+<header v-clickaway="handleClickaway" class="c-header shadow-2">
+    <router-link to="/" class="c-header__logo-container u-flex--left">
+        <div class="c-header__logo-image-container">
+            <img
+                alt="Telos EVM logo"
+                src="~assets/evm_logo.png"
+                width="32"
+            >
+            <div v-if="isTestnet" class="c-header__testnet-indicator">
+                Testnet
+            </div>
+        </div>
+
+        <span class="c-header--logo-text">Teloscan</span>
```

Comment:
I see that you are using BEM correctly, but in this case maybe you wanted to use underscores because this class refers to an element (logo-text) inside the block (c-header) and not to a modifier. Does it make sense?

```suggestion
        <span class="c-header__logo-text">Teloscan</span>
```